servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -481,6 +583,35 @@ def _get_scattered_task_status(shards):
             return status
 
 
+def _convert_to_attempt(item):
+    attempt = IndividualAttempt(
+        execution_status=task_statuses.cromwell_execution_to_api(
+            item.get('executionStatus')),
+        attempt_number=item.get('attempt'),
+        call_cached=_is_call_cached(item.get('callCaching')),
+        stdout=item.get('stdout'),
+        stderr=item.get('stderr'),
+        call_root=item.get('callRoot'),
+        inputs=item.get('inputs'),
+        outputs=item.get('outputs'),
+        start=_parse_datetime(item.get('start')),
+        end=_parse_datetime(item.get('end')))
+
+    if item.get('failures'):
+        attempt.failure_messages = [
+            f.get('message') for f in item.get('failures')
+        ]
+
+    return attempt
+
+
+def _is_call_cached(metadata):
+    if metadata.get('hit') is not None:
+        return metadata.get('hit')
```

Comment:
this is optional since getter is pretty fast, but having:
```
hits = metadata.get('hit')
if hits:
    return hits
```
helps if it was a slower function call